packages/@aws-cdk/aws-sns/README.md

```diff
@@ -95,6 +95,20 @@ topic.addSubscription(new subs.LambdaSubscription(fn, {
 }));
 ```
 
+### Example of Firehose Subscription
```

Comment:
`~` (Nit) This is okay, if we're going to leave this implementation as the basics only. I'd greatly prefer we document it as part of a `FirehoseSubscription` in the `aws-sns-subscriptions` README instead though.